sysdig/sysdig_clients.go

```diff
@@ -159,6 +186,26 @@ func getIBMSecureVariables(data *schema.ResourceData) (*ibmVariables, error) {
 	return getIBMVariables("secure", data)
 }
 
+func (c *sysdigClients) Configure(ctx context.Context, d *schema.ResourceData) {
+	c.ctx = ctx
+	c.d = d
+}
+
+func (c *sysdigClients) AddCleanupHook(cleanupHook func(context.Context, SysdigClients) error) {
+	c.mu.Lock()
+	c.cleanupHooks = append(c.cleanupHooks, cleanupHook)
```

Comment:
What would happen if we add multiple cleanup hooks of the same type, in this case the sendPoliciesToAgents?